Tests/StreamChatTests/Database/DTOs/CurrentUserDTO_Tests.swift

```diff
@@ -290,4 +290,47 @@ final class CurrentUserModelDTO_Tests: XCTestCase {
         XCTAssertEqual(true, loadedCurrentUser.privacySettings.readReceipts?.enabled)
         XCTAssertEqual(true, loadedCurrentUser.privacySettings.typingIndicators?.enabled)
     }
+    
+    func test_deletingCurrentUser() throws {
```

Comment:
Covering the cascade deletion change. If the rule is cascade for mutedUsers, then this test fails and the channel's member ids are: "", "current_user_id"